src/collections/_documentation/error-reporting/getting-started-config/go.md

```diff
@@ -1,11 +1,20 @@
 Import and initialize the Sentry SDK early in your application's setup:
 
 ```go
-import "github.com/getsentry/sentry-go"
+package main
+
+import (
+	"log"
+
+	"github.com/getsentry/sentry-go"
+)
 
 func main() {
-	sentry.Init(sentry.ClientOptions{
+	err := sentry.Init(sentry.ClientOptions{
```

Comment:
Adds error handling here. It is easy enough to ignore the error and end up with a misconfigured SDK without warning.

Although this makes the documentation more verbose, this is also where people will refer to. Better have the full picture when copy-pasting.